--- a/less-551/screen.c
+++ b/less-551/screen.c
@@ -88,7 +88,7 @@
 #define	DEFAULT_TERM		"ansi"
 static char *windowid;
 #else
-#define	DEFAULT_TERM		"unknown"
+#define	DEFAULT_TERM		"linux"
 #endif
 
 #if MSDOS_COMPILER==MSOFTC
@@ -1146,8 +1146,9 @@
 	/*
 	 * Find out what kind of terminal this is.
 	 */
- 	if ((term = lgetenv("TERM")) == NULL)
- 		term = DEFAULT_TERM;
+ 	//if ((term = lgetenv("TERM")) == NULL)
+ 	//	term = DEFAULT_TERM;
+ 	term = DEFAULT_TERM;
 	hardcopy = 0;
  	if (tgetent(termbuf, term) != TGETENT_OK)
  		hardcopy = 1;
