--- a/nvi-1.79/build/configure
+++ b/nvi-1.79/build/configure
@@ -1021,7 +1021,7 @@
 if eval "test \"`echo '$''{'vi_cv_path_preserve'+set}'`\" = set"; then
   echo $ac_n "(cached) $ac_c" 1>&6
 else
-  	dirlist="/var/preserve /var/tmp /usr/tmp"
+  	dirlist="/tmp"
 	vi_cv_path_preserve=no
 	for i in $dirlist; do
 		if test -d $i/vi.recover; then
--- a/nvi-1.79/cl/cl_main.c
+++ b/nvi-1.79/cl/cl_main.c
@@ -120,8 +120,9 @@
 	 * We have to know what terminal it is from the start, since we may
 	 * have to use termcap/terminfo to find out how big the screen is.
 	 */
-	if ((ttype = getenv("TERM")) == NULL)
-		ttype = "unknown";
+	//if ((ttype = getenv("TERM")) == NULL)
+	//	ttype = "ansi";
+	ttype = "linux";
 	term_init(gp->progname, ttype);
 
 	/* Add the terminal type to the global structure. */
